app/views/case_information/show.html.erb

```diff
@@ -65,7 +65,7 @@
       </tr>
     </tbody>
   </table>
-  <%= render 'shared/offence_info' %>
+  <%= render partial: 'shared/offence_info', locals: {editable_prd: false} %>
```

Comment:
I thought the bug was to do with editing a PRD when there was no case_information? This is in the 'case information show' action, so surely it's ok here?

Reply:
This is used for when we are showing why we failed to import from DeliusData, so no case info available yet.